backend/api/authentication.py

```
@@ -0,0 +1,67 @@
+                    role=role
+                )
+                
+                # Create relevant profile based on role
+                if role == 'company':
+                    from .models import CompanyProfile
+                    CompanyProfile.objects.create(user=user)
+                elif role == 'user':
+                    from .models import ViewerProfile
+                    ViewerProfile.objects.create(user=user)
+                
+            return (user, None)
```

Ruff (0.8.2)

Consider moving this statement to an `else` block

(TRY300)